JavaScript/Topics/Aync And Await/README.md

```diff
@@ -0,0 +1,156 @@
+
+## Javascript Async And Await
+
+JavaScript is always synchronous and single-threaded that provides the event loops. The event loops enable us to queue up an activity. This activity will not happen until the loops become available after the program that queued the action has completed the execution. However, our program contains a large number of functionalities, which causes our code to be asynchronous. 
+The Async/Await functionality is one of them. Async/Await is an extension of promises that we get as language support.
+
+**Async**
+
+A Async function is a function declared with **Async** keyword wic means it will run in the backround while performing the code inside when the function is done, it automatically returns a promise.
```

Comment:
**An** Async function is a function declared with **the**  **Async** keyword **which** means it will run in the **background** while performing the code inside when the function is done, it automatically returns a promise.
@7sakshi7 Please correct some grammar and spelling errors and after that, I think you are good to go. Cheers!

Reply:
@s4-2 Done